deploy/helm/templates/generated/Deployment-clickhouse-operator.yaml

```diff
@@ -113,6 +113,7 @@ spec:
                   resource: limits.memory
             {{ with .Values.operator.env }}{{ toYaml . | nindent 12 }}{{ end }}
           resources: {{ toYaml .Values.operator.resources | nindent 12 }}
+          securityContext: {{ toYaml .Values.operator.containerSecurityContext | nindent 12 }}
```

Comment:
it will not work
you tried change templates/generated
look to https://github.com/Altinity/clickhouse-operator/blob/0.22.0/dev/generate_helm_chart.sh

you need add empty securityContext via yq